runtime/doc/usr_28.txt

```diff
@@ -376,7 +376,7 @@ More about folding by expression in the reference manual: |fold-expr|
 *28.9*	Folding unchanged lines
 
 This is useful when you set the 'diff' option in the same window.  The
-|vimdiff| command does this for you.  Example: >
+|-d| argument does this for you.  Example: >
```

Comment:
I could say `command line option`, to avoid mixing with vim's notion of internal options (e.g. `option`).